packages/rpc/src/trust-client/models/PriceInfo.ts

```diff
@@ -0,0 +1,26 @@
+import { Transform, Type } from 'class-transformer';
+import BigNumber from 'bignumber.js';
+
+export class Blockchain {
```

Comment:
do we need this class `Blockchain` . ?

Reply:
It's just migrated from the platform here. It's not used. Let's remove it